src/zimwriterfs/zimwriterfs.cpp

```diff
@@ -316,6 +323,7 @@ void parse_args(int argc, char** argv)
     usage();
     exit(1);
   }
+  }
```

Comment:
Shouldn't we check the directoryPath and zimPath even if we have `dontCheckArgs` ?

Reply:
Fixed. Also noticed and fixed an error in the short option corresponding to `--dont-check-arguments` (I had initially used `-g`, then changed it to `-B` but forgot to update the call to `getopt_long()`).